packages/node_modules/@webex/internal-plugin-mercury/src/socket/socket-base.js

```diff
@@ -193,7 +195,7 @@ export default class Socket extends EventEmitter {
       const WebSocket = Socket.getWebSocketConstructor();
 
       this.logger.info('socket: creating WebSocket');
-      const socket = new WebSocket(url);
+      const socket = new WebSocket(url, options);
```

Comment:
I'm not sure how the reflect.defineProperty is supposed to work above, but we found that the options had to be passed to the websocket constructor for the agent to take effect.